src/pages/test-report/components/competency/competency.html

```diff
@@ -1,9 +1,27 @@
-<button #competencyButton>
-  <fault-counter id="drivingFaults" [count]="faultCount"></fault-counter>
-  <span id='competencyLabel'>{{ getLabel() }}</span>
-  <div>
+<button #competencyButton (click)="togglePromote()" [class.promoted]="promoted">
```

Comment:
I feel that we should have 2 buttons, the competency button  and a specific one for manoeuvres. It seems like this one button is going to be getting complicated with fault counts, tick indicators, popups, different styled labels and all the logic around them all